recipes-ni/ni-rtfeatures/files/init.d/handle_cpld_ip_reset

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+IP_RESET_FILE="/dev/nirtfeatures/ip_reset"
+
+IP_RESET_REQUESTED=""
+if [ -e "$IP_RESET_FILE" ]; then
+	grep -q '1' $IP_RESET_FILE && IP_RESET_REQUESTED="y"
+else
+	[ "${VERBOSE}" != "no" ] && echo "Unable to find ip_reset at normal spot."
+fi
+
+if [ "$IP_RESET_REQUESTED" == "y" ]; then
+	mkdir -p /tmp/ip_reset_dir
+	/usr/local/natinst/bin/ninetcfgutil pulldefault -d /tmp/ip_reset_dir -g primary -g secondary
```

Comment:
I can't recall if `$PATH` is valid this early in boot. But if it is, I think we generally prefer to let `PATH` locate commands, in cases where the command binary isn't owned by the initscript package. Can you change this to just `ninetcfgutil ...`?

Reply:
Relying on `$PATH` works, so update to call `ninetcfgutil ` instead.